src/main/java/com/zufar/onlinestore/cart/endpoint/CartEndpoint.java

```diff
@@ -0,0 +1,38 @@
+package com.zufar.onlinestore.cart.endpoint;
+
+import com.zufar.onlinestore.cart.api.CartApi;
+import com.zufar.onlinestore.cart.dto.ShoppingSessionDto;
+import com.zufar.onlinestore.cart.dto.UpdateProductsQuantityInShoppingSessionItemRequest;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping(value = CartEndpoint.CART_URL)
+public class CartEndpoint {
+
+    public static final String CART_URL = "/api/v1/cart";
+
+    private final CartApi cartApi;
+
+    @PatchMapping
+    @ResponseBody
+    public ResponseEntity<ShoppingSessionDto> updateProductsQuantityInShoppingSessionItem(@RequestBody @Valid final UpdateProductsQuantityInShoppingSessionItemRequest request) {
+        log.info("Received request to update productsQuantity in shoppingSession item with id - {}.", request.shoppingSessionItemId());
```

Comment:
Please fill log message by format: 
"methodName: executionStatus : valuableArguments"

It will be easier to understand what's going on in Kibana logs and also convenient to look for logs associated with a particular method.

Examples:

log.info("update products quantity in shopping session item: received request:  shoppingSession item with id - {}.", request.shoppingSessionItemId())
 
log.info("update products quantity in shopping session item: updated:  shoppingSession item with id = {}"request.shoppingSessionItemId())

log.info("update products quantity in shopping session item: failed:  shoppingSession item with id = {}"request.shoppingSessionItemId())

Reply:
Fixed